dev/NumberBox/NumberBox.cpp

```diff
@@ -421,6 +421,8 @@ void NumberBox::OnNumberBoxScroll(winrt::IInspectable const& sender, winrt::Poin
             {
                 StepValue(-SmallChange());
             }
+            // Only set as handled when we actually changed our state.
+            args.Handled(true);
```

Comment:
What if the StepValue call from above didn't do anything? For instance, what if we are at the max value and scroll up. It might be confusing for the user to be scrolling the mouse wheel while in a scroll viewer and not see anything happen on screen, which could be the case at the max or min values or when ValidateInput fails.

Reply:
That is definitely a problem. What would be the expected behavior from a UX standpoint? I have just checked in Edge Beta and Chrome, and the behavior in browsers seems to be that the page does not scroll when even when the number input has reached its maximum or minimum.

Would it be better to align with web browsers or is it better to go our own way on this matter?

Reply:
good question, @SavoySchuler @jevansaks @teaP

Reply:
I think we should handle it always, not only if the value changed. It seems confusing to have the feedback that you reached the maximum value be that the page starts scrolling.

Reply:
I think that makes sense, however a counter example is this is the exact feedback you get when scrolling nested scroll viewers. If you think about this case as scrolling the numberbox then maybe the extension of scroll viewers behavior says we should not handle?

Reply:
Is that behavior always appropriate? The difference here is that we're only handling mouse wheel when the control has focus. ScrollViewer needs that chaining because it'll scroll no matter what if the mouse pointer is over it.

Reply:
Probably not, I just wanted to make sure we thought about it.